modules/acm_sku/src/Entity/SKU.php

```diff
@@ -757,4 +757,20 @@ public function clearStockCache() {
     }
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
+    $cache_tags = parent::getCacheTags();
+    /** @var \Drupal\acm_sku\AcquiaCommerce\SKUPluginBase $plugin */
+    $plugin = $this->getPluginInstance();
+    // Get parent skus(if any) for the sku.
+    $parent_skus = array_values($plugin->getAllParentSkus($this->getSku()));
+    // Prepare cache tags of parent sku.
+    $parent_skus = array_map(function($parent_sku) { return 'acm_sku:'. $parent_sku; }, $parent_skus);
+
+    // @Todo: Add the tags of the display node as well.
+    return Cache::mergeTags($cache_tags, $parent_skus);
```

Comment:
this should be in getcachetagstoinvalidate, we don't want these cache tags all the time

Reply:
getCacheTags() internally calls getcachetagstoinvalidate().

Reply:
@joshirohit100 will Drupal use getCacheTags to invalidate when entity is saved?

Reply:
check invalidateTagsOnSave in docroot/core/lib/Drupal/Core/Entity/Entity.php

Reply:
adding missing use statement in https://github.com/acquia/commerce-manager/pull/126